ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -496,6 +512,9 @@ export default defineComponent({
 
     })
 
+    const convertedTooltipText: string = 'The records for this registration were converted from paper to digital ' +
```

Comment:
Since this is a static text, why not put it straight into the template?

Reply:
I could, just thought it might look a little cleaner as the template is already pretty crowded. What do you think?

Reply:
A `<template>` is meant to hold HTML structure, content, etc. anything that the user interacts with. `<script>` is for the dynamic stuff, it is used to define logic/behaviour of the component. It does sometimes include dynamic content as it might be cleaner. But for static, it should be in the template.

Reply:
Definitely agree with Dima here. 
There is a place for declaring string values in the script tho, a url for example. 

In this case, the static text can go right in :)

Reply:
Gotcha, thanks guys. Changes made